templates/io.html

```diff
@@ -1,76 +1,138 @@
+        <a href="subject/18">Plant Biology</a><br>
+        &emsp;
+        <a href="subject/19">Developmental Biology</a><br>
+        &emsp;
+        <a href="subject/20">System Biology</a><br>
+        &emsp;
+        <a href="subject/21">Biochemistry</a><br>
+        &emsp;
+        <a href="subject/22">Biophysics and Computational Biology</a><br>
+        <a href="subject/4">Computer Sciences</a><br>
+        &emsp;
+        <a href="subject/23">Machine Learning</a><br>
+        &emsp;
+        <a href="subject/24">Computational Biology</a><br>
+        &emsp;
+        <a href="subject/25">Computational Complexity</a><br>
+        &emsp;
+        <a href="subject/26">Computational Linguistics</a><br>
+        <a href="subject/5">Statistics</a><br>
+        &emsp;
+        <a href="subject/27">Applied Statistics</a><br>
+        &emsp;
+        <a href="subject/28">Mathematical Statistics</a><br>
+        <a href="subject/6">Mathematics</a><br>
+        &emsp;
+        <a href="subject/29">Graphy Theory</a><br>
+        &emsp;
+        <a href="subject/30">Number Theory</a><br>
+        <a href="subject/7">Medical Sciences</a><br>
+        &emsp;
+        <a href="subject/31">Breast Cancer</a><br>
+        &emsp;
+        <a href="subject/32">Ebola Virus</a><br>
+        &emsp;
+        <a href="subject/33">Epidemics</a><br>
+        &emsp;
+        <a href="subject/34">Dermatology General</a><br>
+        &emsp;
+        <a href="subject/35">Immunity</a><br>
+        &emsp;
+        <a href="subject/36">Obesity</a><br>
+
+    </div>
+    <div style="background-color:aquamarine;height:1423px;width:700px;float: left;box-sizing: border-box;">
+   <h1 style="background-color: rgb(0, 56, 37);color:azure;">Top5 Articles</h1>
+   &emsp;
+   <a href="article/1">Lab 3</a><br>
+   &emsp;
+   <a href="article/2">Draft 1</a><br>
+   
+   &emsp;
+   <a href="article/3">my Article</a><br>
+   
+   &emsp;
+   <a href="article/5">sweet texts</a><br>
+   &emsp;
+   <a href="article/7">Astronaut </a><br>
+   &emsp;
+   <h1 style="background-color: rgb(0, 56, 37);color:azure;">All Articles</h1>
+   &emsp;
+   <a href="article/1">Lab 3<span style="float: right;">2019-11-09 12:00:23</span></a><br>
+   &emsp;
+   <a href="article/2">Draft 1<span style="float: right;">2019-11-09 13:01:09</span></a><br>
+   
+   &emsp;
+   <a href="article/3">my Article<span style="float: right;">2020-03-18 06:41:50</span></a><br>
+   
+   &emsp;
+   <a href="article/5">sweet texts<span style="float: right;">2020-03-19 05:57:19</span></a><br>
+   &emsp;
+   <a href="article/7">Astronaut<span style="float: right;">2020-03-19 07:28:16</span> </a><br>
+   &emsp;
+   <a href="article/8">Spaceships<span style="float: right;">2020-03-19 07:29:33</span></a><br>
+   &emsp;
+   <a href="article/9">Venus<span style="float: right;">2020-03-19 07:30:34</span></a><br>
+   
+   &emsp;
+   <a href="article/10">The Article<span style="float: right;">2020-04-19 20:30:33</span></a><br>
```

Comment:
@CZH-Gaizka The article is hard-coded, not generated dynamically from database query? -Hui

Reply:
All the title and time are shown by jinja frame. By the way, I changed the color style of this page and I upload a new screenshot to show the final result.